key_point[] kps
descriptors des
uint8 autor
float32 roll
float32 pitch
